src/docs/background/04-getting-started/index.md

```diff
@@ -11,27 +11,33 @@ ingress:
 published: true
 ---
 
-## Introducing Flex
+## [Introducing Flex](/background/introducing-flex/)
 
 Introduction to the three main components of Flex: your marketplace user
 interface, the Marketplace API, and Console. We explain the role each of
 these components plays and how they help you to build a unique
 marketplace platform. [Read more »](/background/introducing-flex/)
 
-## How to know if Flex is the right fit for your marketplace idea
+---
```

Comment:
I don't think it makes sense to require all article authors to add these. This will only make the content inconsistent as people forget adding these lines. Could we just style the article `h2` elements to have a top border?

Reply:
@kpuputti hmm, yeah maybe having a border-top for H2 (excl. the first H2 on the page) would make it easier for the content writers - I'll create the classes and remove the lines! 🍬